### PR TITLE
upgrade Personalizer RL.NET to 0.2.0

### PR DESCRIPTION
**Pull request by homezcx**
[upgrade Personalizer RL.NET to 0.2.0](https://github.com/Azure/azure-sdk-for-net/commit/22e5726b42dad6c973bd5a97090ac92746faa808)

### CODE REVIEW

**Review by orenmichaely — APPROVED** [review]

**Review by jackgerrits — APPROVED** [review]

File: eng/Packages.Data.props
Comment (weshaggard) [inline eng/Packages.Data.props:109]:
```suggestion
    <PackageReference Update="RL.Net" Version="0.2.0"/>
```

Is this package dependency approved by ArchBoard? /cc @tg-msft @KrzysztofCwalina

**Review by weshaggard — CHANGES_REQUESTED** [review]
We should not merge this change until @tg-msft approves this dependency from ArchBoard prospective. We also might want to limit the scope of this dependency to only allow it only for Personalizer and not all the other libraries.

**Comment by homezcx** [comment]
> We should not merge this change until @tg-msft approves this dependency from ArchBoard prospective. We also might want to limit the scope of this dependency to only allow it only for Personalizer and not all the other libraries.

Reverted the change and only update the nuget version in existing project files.

File: sdk/personalizer/Azure.AI.Personalizer/src/Azure.AI.Personalizer.csproj
Comment (weshaggard) [inline sdk/personalizer/Azure.AI.Personalizer/src/Azure.AI.Personalizer.csproj:24]: If you are doing this directly in the project it will need to be VersionOverride. I definitely think this is better then moving this preview dependency to the common props file but we still do need to get this dependency reviewed and approved and we should avoid shipping any stable versions of this package with this prerelease dependency.

Comment (homezcx) [reply sdk/personalizer/Azure.AI.Personalizer/src/Azure.AI.Personalizer.csproj]: Thanks. Updated. Yes. Let's wait for the dependency review.

**Review by weshaggard — APPROVED** [review]

File: sdk/personalizer/Azure.AI.Personalizer/src/Azure.AI.Personalizer.csproj
Comment (homezcx) [reply sdk/personalizer/Azure.AI.Personalizer/src/Azure.AI.Personalizer.csproj]: @tg-msft -> Could you help check the comments to review this dependency?

Comment (tg-msft) [reply sdk/personalizer/Azure.AI.Personalizer/src/Azure.AI.Personalizer.csproj]: When we last discussed this in https://github.com/Azure/azure-sdk/issues/3877#issuecomment-1043638770, I believe the conclusion was you'd bring your dependency back to the Arch Board when ready because there were open questions like how to make this pay-for-play, what this looks like across our platform matrix, whether this works on other platforms, etc.  That still blocks your GA release per comments in https://apiview.dev/Assemblies/Review/35ddf25e338f4bb88d114ab43933c003.

That said, I have no concerns with folks updating the package in the repo to continue working on it though.  As @weshaggard suggested, this should definitely be scoped to just Personalizer.  (I'm not sure why anyone else would _want_ to take a dependency on it, but I'd rather we get a heads up if they tried.)

**Pull request merged**
